apps/coscrad-frontend/src/coscrad-theme-provider.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { screen } from '@testing-library/react';
+
+import { CoscradThemeProvider } from './coscrad-theme-provider';
+import { renderWithProviders } from './utils/test-utils';
+import { getDummyConfigurableContent } from './utils/test-utils/get-dummy-configurable-content';
+
+describe('CoscradThemeProvider', () => {
+    it('should render successfully', () => {
```

Comment:
This is not the approach you want to take. What you need to do is for each component that leverages the theme,
render **that component** with a dummy config where you explicitly set just the relevant property. Then check that the property is indeed used by querying the DOM and finding the appropriate attribute value.